### PR TITLE
Add 404 Page 

### PR DESCRIPTION
**Pull request by MrNickCasey**
Instead of redirecting to the homepage on an invalid URL, let's send the user to a well-designed 404 page.

### CODE REVIEW

**Comment by MrNickCasey** [comment]
[Create 404 Page](https://trello.com/c/ZFn9kPIM/212-create-404-page)

File: src/pages/invalid/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Navbar, 
+    Button 
+} from 'components';
+import styles from './index.module.css';
+
+class Forgot extends React.Component {
+  render () {
+    return (
+      <div className={styles.invalidPage}> 
+        <Navbar />
```
Comment (FemmeQueenInSTEM) [inline src/pages/invalid/index.js]: You should add a helmet here to change the title of the page
```javascript
import {Helmet} from 'react-helmet';
// ... in your render function
<Helmet>
   <title>Page not found</title>
</Helmet>

```

**Review by walsquared — CHANGES_REQUESTED** [review]
Looks DOPE, add this as a final touch

**Comment by walsquared** [comment]
Wave one of applications is about to end, so I had to do this myself.

**Pull request merged**